include/app/session.hpp

```diff
@@ -140,8 +140,8 @@ class Session {
         static void symbolize(QAbstractButton *button);
 
         Forge(Arbiter &arbiter);
-        void iconize(QString name, QAbstractButton *button, uint8_t size, bool dynamic = false) const;
-        void iconize(QString name, QString alt_name, QAbstractButton *button, uint8_t size, bool dynamic = false) const;
+        void iconize(QString name, QAbstractButton *button, uint8_t size, bool dynamic = true) const;
+        void iconize(QString name, QString alt_name, QAbstractButton *button, uint8_t size, bool dynamic = true) const;
```

Comment:
As I said before, having the icons be dynamic is very expensive and should be done sparingly, certainly not by default. Most of the icons are not supposed to be colorized.